flink-end-to-end-tests/test-scripts/test_batch_allround.sh

```diff
@@ -24,31 +24,13 @@ TEST_PROGRAM_JAR=${END_TO_END_DIR}/flink-dataset-allround-test/target/DataSetAll
 echo "Run DataSet-Allround-Test Program"
 
 # modify configuration to include spilling to disk
-cp $FLINK_DIR/conf/flink-conf.yaml $FLINK_DIR/conf/flink-conf.yaml.bak
 echo "taskmanager.network.memory.min: 10485760" >> $FLINK_DIR/conf/flink-conf.yaml
 echo "taskmanager.network.memory.max: 10485760" >> $FLINK_DIR/conf/flink-conf.yaml
 
 backup_config
```

Comment:
This doesn't look right, why are we creating a backup after modifying the config? Also, we never seem to restore it anywhere.

Reply:
You're right, this command must be moved above the two `echo` lines. The config will be reverted by the `test-runner-common.sh`. With FLINK-10094 the backup moved to this script as well but it is not part of `release-1.5`. I will move this command before the two echos.